src/components/Publish/Preview/index.tsx

```diff
@@ -34,8 +33,10 @@ export default function Preview(): ReactElement {
         owned: false,
         validOrderTx: ''
       }
-      setAccessDetails(accessDetails)
+      // TODO: add Property 'accessDetails' to type 'DDO' on ocean.js. see <Asset>
```

Comment:
no, `accessDetails` is only on market. it is part of `AssetExtended`

Reply:
ok, restored 👍🏽